packages/office-ui-fabric-react/src/components/Button/DefaultButton/DefaultButton.Props.ts

```diff
@@ -0,0 +1,8 @@
+/* tslint:disable:no-unused-variable no-unused-imports */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable no-unused-imports*/
+import { IButtonProps } from '../ButtonBase/ButtonBase.Props';
+
+export interface IDefaultButtonProps extends IButtonProps {
```

Comment:
There's no need for this file/interface, right? DefaultButton has no additional props to add.